### PR TITLE
Teams and collectives intro edits

### PR DESCRIPTION
**Pull request by davidozog**
Minor edits that make the Teams and Collectives introduction texts more consistent/concise.

### CODE REVIEW

File: content/collective_intro.tex [hunk 1]
```diff
@@ -27,21 +27,17 @@ \subsubsection*{Team-based collectives}
 or \LibConstRef{SHMEM\_TEAM\_INVALID} is passed to a team-based collective
 routine, the behavior is undefined.
 
-Team objects encapsulate the per \ac{PE} system resources required to complete
-team-based collective routines.
-All \openshmem teams-based collective calls are blocking routines which may use those
-system resources. On completion of a team-based collective call, the \ac{PE} may
-immediately call another collective on that same team without any other intervening
-synchronization across the team.
+All \openshmem teams-based collective calls are blocking routines.  On
```
Comment (nspark) [inline content/collective_intro.tex]:
```suggestion
All \openshmem teams-based collective operations are blocking routines.  On
```

File: content/collective_intro.tex [hunk 2]
```diff
@@ -27,21 +27,17 @@ \subsubsection*{Team-based collectives}
...
+completion of a team-based collective call, the \ac{PE} may immediately call
```
Comment (nspark) [inline content/collective_intro.tex]:
```suggestion
return from a team-based collective call, the \ac{PE} may immediately call
```

File: content/collective_intro.tex [hunk 3]
```diff
@@ -27,21 +27,17 @@ \subsubsection*{Team-based collectives}
...
+another collective on that same team without any other intervening
```
Comment (nspark) [inline content/collective_intro.tex]:
```suggestion
another collective operation on that same team.
```

File: content/collective_intro.tex [hunk 4]
```diff
@@ -27,21 +27,17 @@ \subsubsection*{Team-based collectives}
...
+synchronization across the team.  Team-based collectives must occur in the same
```
Comment (nspark) [inline content/collective_intro.tex]:
```suggestion
Team-based collectives must occur in the same
```

**Review by nspark — COMMENTED** [review]
@davidozog I'm suggesting some small rephrasings. What do you think? (Too bad GitHub doesn't support multiline suggestions.)

**Comment by davidozog** [comment]
@nspark They're good changes, thanks!

**Review by nspark — APPROVED** [review]

**Pull request merged**